tests/test_black.py

```diff
@@ -1679,6 +1679,18 @@ def test_reformat_one_with_stdin_and_existing_path(self) -> None:
             # __BLACK_STDIN_FILENAME__ should have been stripped
             report.done.assert_called_with(expected, black.Changed.YES)
 
+    def test_reformat_one_with_stdin_empty(self) -> None:
+        with patch("sys.stdout", open(os.devnull, "w")):
+            # Patch `sys.stdout` to null device lest
+            # `format_stdin_to_stdout` detaches stdout at the end and causes
+            # "ValueError: I/O operation on closed file" in subsequent tests
+            black.format_stdin_to_stdout(
```

Comment:
Can you assert that it reformats to the empty string? Maybe by setting stdout to a StringIO object.

Reply:
Sure! Here it is.